modules/ROOT/pages/business-groups.adoc

```diff
@@ -0,0 +1,111 @@
+* View the parent business group's client ID and client secret
+
+When you create a business group under another business group, only the redistributable entitlements, such as VPCs and load balancers, that were assigned to the parent business group can be allocated to the child business group.
+
+=== Create a Business Group
+
+When you create a new business group under the top-level business group, all the current users with the *Organization Administrators* role in the top-level business group appear in the list of users in the newly created business group. 
+
+. In the Anypoint Platform *Access Management* page, in the left menu, click *Organization*.
+. Click the blue plus sign next to the organization name or any of the business groups in the tree diagram.
++
+image::organization-20885.png[]
++
+. In the dialog box, enter:
+  * *Business Group name*: Name for the new business group.
+  * *Owner*: Assign an existing user, with the *Organization Administrators* role, as the business group owner.
+  * Select *Owner can create Business Groups* to allow the owner to create child business groups under this business group.
+  * Select *Owner can create environments* to allow the owner to create environments within this business group.
+  * You can assign some or all of the redistributable resources (vCores, VPCs, and so on) that your organization owns to an individual business group. This ensures that the resources are used by the CloudHub deployments that belong to the business group. You can assign the resources when you create the business group, or edit these settings later.
+  * *Enable CloudHub global deployment*: This option is available only if global deployment is enabled on the parent business group. When global deployment is enabled, the region is auto-populated according to the default region you specified.  
+* *Static IPs*: This option is available only if the parent business group has static IPs assigned to it. This option enables the use of static IP addresses. 
+. Click *Add Business Group*. +
+The business group appears in the hierarchy, under the master organization.
+
+image::organization-9fa5f.png[]
+
+[NOTE]
+--
+Allocating redistributable resources to a business group makes those resources available only to that business group, which makes them unavailable to the parent organization.
+--
+
+=== Create a Child Business Group
+
+You can create hierarchical levels of business groups to provide more control over access. 
+
+. Click the blue *+* icon on the left of the business group in which to create the child. 
+. In the *Add Business Group* dialog, enter a *Business Group name* and *Owner*, then select from these options: +
+* *Owner can create Business Groups*: The owner you assign to the business group can create child business groups within the business group for which they are the owner. 
+* *Owner can create environments*: The owner you assign can create environments within the business group. 
```

Comment:
We should not use "owner" as an authorization mechanism. Let's reword to "organization administrators can create environments within the business group"